guide/payments/coin-selection.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: guide
+title: Coin Selection
+nav_order: 1
+parent: Payments
+permalink: /guide/payments/coin-selection/
+---
+
+{% include picture.html
+
+  image = "/assets/images/payments/coin-selection-header.png"
+
+  retina = "/assets/images/payments/coin-selection-header@2x.png"
+
+  mobile = "/assets/images/payments/coin-selection-header.png"
+
+  mobileRetina = "/assets/images/payments/coin-selection-header@2x.png"
+
+  alt-text = ""
+
+  width = 1600
+
+  height = 900
+
+  layout = "full-width"
+
+%}
+
+# Coin Selection
+
+## Intro
+
+Coin selection is the process of choosing which of your Bitcoins to spend when creating and approving a spending transaction. To understand coin selection a little better, we must first grasp the concept of Unspent Transaction Outputs (UTXOs).
+
+A UTXO is the amount of digital currency remaining after a bitcoin transaction has been executed - essentially, the unspent change left after someone completes a transaction, similar to the change someone receives after conducting a cash transaction at the store. They are processed continuously and are responsible for beginning and ending each bitcoin transaction. They might sound complicated, but they really are quite simple. 
+
+How does this relate to coin selection, you might ask? Because UTXOs allow for transactions to be conducted using miltiple fractions of bitcoin that do not all come from a single previous transaction. Instead, multiple fractions of bitcoin are retrieved by the algorithm to fulfill a spending request. For example, a purchase worth 1 bitcoin may retrieve 0.6 BTC from one byte and 0.4 BTC from another. Change from each of these fractions is then sent to the UTXO database to be spent at a later date.
+
+For example, my wallet balance is 1 bitcoin (BTC). If I have recently mined a block, this could be a fresh whole bitcoin. However, it is most likely going to be an accumulation of UTXOs (change) from previous bitcoin transactions. So, a balance of 1 BTC is more likely to comprimise of:
```

Comment:
Thank you @pavlenex for all these comments. This is just a draft version for now but needs review, so thanks for the feedback. As for the UTXO reference, I agree, it would save a lot of time not having to explain this again and repeat information

Reply:
Ah okay @barefoot-88, I wasn't aware this is still a draft, please let me know once it's ready for a review, happy to take a better look when it's ready.

Reply:
Will do, thanks!